lldb/source/Plugins/Language/Swift/SwiftArray.cpp

```diff
@@ -376,8 +376,10 @@ SwiftArrayBufferHandler::CreateBufferHandler(ValueObject &valobj) {
     if (handler && handler->IsValid())
       return handler;
     return nullptr;
-  } else if (valobj_typename.startswith("Swift.ArraySlice<")) {
-    // Swift.ArraySlice
+  } else if (valobj_typename.startswith("Swift.ArraySlice<") ||
+             (valobj_typename.startswith("Swift.Array<") &&
+              valobj_typename.endswith(">.SubSequence"))) {
+    // ArraySlice or Array<T>.SubSequence, which is a typealias to ArraySlice.
```

Comment:
Could this fail on a type that is `Swift.Array<Swift.Array<Int>.Subsequence>>`?

Reply:
Ah.. no. Because it's endswith.

Reply:
right, I believe it's safe.